ci/jenkins/test.sh

```diff
@@ -169,15 +169,15 @@ function export_govc_env_var {
 
 function clean_antrea {
     echo "====== Cleanup Antrea Installation ======"
-    clean_ns "monitoring"
-    clean_ns "antrea-ipam-test"
-    clean_ns "antrea-test"
+    clean_ns "monitoring" || true
```

Comment:
clean_ns already ignores some delete errors, why it needs to futher ignore errors here?

Reply:
In flexible-ipam testbed, the k8s may not be deployed at this moment, thus we need to ignore the error.